lib/scope.js

```diff
@@ -277,47 +277,47 @@ function defineScope(cls, targetClass, name, params, methods, options) {
 
   cls['__get__' + name] = fn;
 
-  var fn_create = function () {
+  var fnCreate = function() {
     var f = this[name].create;
     f.apply(this[name], arguments);
   };
```

Comment:
Makes sense. But then we could still do:

```
function createFn() {
}

cls['__create__' + name] = createFn;
```
ROI is low here anyways, I hate to say it but I guess we follow "if it ain't broke, don't fix it" here. ;)